big-o-notation/big-o.js

```diff
@@ -0,0 +1,44 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {};                  // 1 * 1 = O(1)
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(2n)
+    i++                             // 2 * n = O(2n)
+  ) {
+    const word = words[i];          // 1 * 1 = O(1)
+    if (!seen[word]) {              // 2 * n = O(2n)
+      seen[word] = true;
+      unique[unique.length] = word; // 3 * n = O(3n)
+    }
+  }
+  return unique;
+} // Big O Notation for uniqueLinear: O(n)
+
+function uniqueQuadratic(words) {
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(2n)
+    i++                             // 2 * n = O(2n)
+  ) {
+    const word = words[i];          // 2 * n = O(2n)
+    let isUnique = true;            // 1 * 1 = O(1)
+    for (
+      let c = 0;                    // 1 * 1 = O(1)
+      c < i;                        // n * n = O(n^2)
+      c++                           // 2 * n = O(2n)
+    ) {
+      const comparing = words[c];   // 2 * n = O(2n)
+      if (comparing === word) {     // n * n = O(n^2)
+        isUnique = false;           // 1 * 1 = O(1)
+      }
```

Comment:
Since all of these instructions are inside of a nested loop, we're dealing with `n^2` for each of them. Every time you nest loops you'll need to multiply every step by an extra `n`, like you did on line 35